litellm/llms/prompt_templates/factory.py

```diff
@@ -556,7 +556,7 @@ def convert_to_anthropic_image_obj(openai_image_url: str):
         )
 
 
-def convert_to_anthropic_tool_result(message: dict) -> str:
+def convert_to_anthropic_tool_result(message: dict) -> dict:
```

Comment:
hey @lazyhope i believe this is used by bedrock too

Since the new api isn't available there - can we have a separate function for handling new api output translation?

Reply:
Sure! Thanks for catching that!

Reply:
Hi @krrishdholakia I am not sure but somehow my code also works using bedrock models, could you please test if bedrock claude models actually throws an error?